### PR TITLE
[UT] SectionSolidHorizontal_1 example unit test files

### PR DESCRIPTION
**Pull request by michelangelo-acca**
# IFC4.3 Unit test

## SectionSolidHorizontal_1
Example files generated for the section solid horizontal scenario. It is a simple alignment (horizontal only and also horizontal plus vertical) with different use of IfcFixedReferenceSweptAreaSolid (splitted along the alignment or as one piece).

Further information in README.md file.

## Review requested
- IfcFixedReferenceSweptAreaSolid

### CODE REVIEW

File: IFC 4.3/SectionSolidHorizontal_1/UT_SectionSolidHorizontal_1.ifc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+#65 = IFCAXIS2PLACEMENT3D(#18, #63, #64);
+#66 = IFCDIRECTION((1., 0., 0.));
+#67 = IFCSHAPEREPRESENTATION(#20, 'Body', 'SweptSolid', (#62));
+#68 = IFCBUILDINGELEMENTPROXY('2g$2U96f58Jw0qMUBrAOmy', #1, 'ballast example', $, $, #69, #73, $, $);
+#69 = IFCLOCALPLACEMENT(#36, #72);
+#70 = IFCDIRECTION((0., 0., 1.));
+#71 = IFCDIRECTION((1., 0., 0.));
+#72 = IFCAXIS2PLACEMENT3D(#18, #70, #71);
+#73 = IFCPRODUCTDEFINITIONSHAPE($, $, (#79));
+#74 = IFCFIXEDREFERENCESWEPTAREASOLID(#27, #77, #45, $, $, #78);
```
Comment (pjanck) [inline IFC 4.3/SectionSolidHorizontal_1/UT_SectionSolidHorizontal_1.ifc:81]: I feel that using ÌfcFixedReferenceSweptAreaSolid` is not the correct choice here. It should rather be the `IfcSectionedSolidHorizontal`.  @TLiebich - opinion?

**Review by pjanck — COMMENTED** [review]
I'm not sure about the use of `IfcFixedReferenceSweptAreaSolid` here. It this stays, then the name of the unit test is VERY misleading and needs to be changed.

File: IFC 4.3/SectionSolidHorizontal_1/UT_SectionSolidHorizontal_1.ifc (hunk 1 above)
Comment (TLiebich) [reply IFC 4.3/SectionSolidHorizontal_1/UT_SectionSolidHorizontal_1.ifc:81]: usually sleepers in a railway would follow the local z-direction (perpendicular to the vertical alignment) - so a fixed reference swept area solid would work (not sure about slabtrack, but I would assume that it has a constant thickness independently of the slope). 

but then, the name of the UC should change.

**Comment by michelangelo-acca** [comment]
This is now outdated. It will be created a new pull request with the updated data.

**Pull request closed without merging**